provisions/roles/cleanup/scanner/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Remove all containers
+  sudo: yes
+  shell: docker rm -f `docker ps -a -q`
+
+- name: Remove all images
+  sudo: yes
+  shell: docker rmi -f `docker images -a -q`
```

Comment:
IMO, this is not required as scanner images gets updated each deployment time. and there is no extra images there(if scanner deletes everything by itself).